content/en/docs/actions.md

```diff
@@ -0,0 +1,203 @@
++++
+title =  "Actions"
+
+
+date = 2021-04-07
+lastmod = 2021-04-07
+
+draft = false  # Is this a draft? true/false
+toc = true  # Show table of contents? true/false
+type = "docs"  # Do not modify.
+
+# Add menu entry to sidebar.
+linktitle = "Actions"
+[menu.docs]
+  parent = "scheduler"
+  weight = 2
++++
+
+### Enqueue
+
+```
+Queues
+QueueMap
+jobsMap
+
+//Scan the Job and initialize the above three data structures
+For job in ssn.Jobs 
+//Filter 1
+found := ssn.Queues[job.Queue]
+existed := queueMap[queue.UID]
+//Filter 2
+if job.PodGroup.Status.Phase == scheduling.PodGroupPending
+found := jobsMap[job.Queue];
+
+
+/*Update Node resource usage*/
+For node in ssn
+  Update Total_nodes
+  Update Used_nodes
+
+/*Repeat the enqueue operation*/
+While(!queues.Empty()){
+
+  /*Enqueue operation terminates out of resources*/
+  if idle.IsEmpty() break
+  //deal with target job , if exists , judege whether it can be
+
+  minReq <--- job
+  idle   <--- node resources
+  if node Adequate resources
+  inqueue := true;
+  if inqueue
+     enqueue
+  Queues.push(queue)
+} 
+
+```
+
+#### Logic
+
+Filter the tasks that meet the requirements and enter the queue to be scheduled.The filtering condition is whether node meets the task's minimum resource requirement minReq.
+
+####  Scence
```

Comment:
%s/Scene/Scenario

Reply:
Filter the tasks that meet the requirements and enter the queue to be scheduled. The filtering condition is whether the node meets the task's minimum resource requirement (minReq).